src/System.Data.Odbc/src/Common/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Data.Common
+{
+    internal static class DbConnectionStringDefaults
```

Comment:
I just did that for the common files in `ProviderBase`, which were exactly the same minus style fixes. Please let me know if you see anything wrong! https://github.com/dotnet/corefx/pull/15646/commits/b37222a0b48bcdbb044d8b14678cf6df775c5730